website/integrations/services/semaphore/index.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Integrate with Semaphore UI
+sidebar_label: semaphore
+---
+
+# Semaphore UI
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Semaphore UI
+
+> Semaphore UI is a modern web interface for managing popular DevOps tools.
+> -- https://semaphoreui.com/
+>
+> This guide explains how to set up Semaphore UI to use authentik as the OAuth provider for the login to the WebGUI.
```

Comment:
```suggestion
> This guide explains how to configure Semaphore UI to use authentik as the OAuth provider for logging in to the Web GUI.
```

Reply:
oh I used it from my previous PR's that I made here :D (take a look @ engomo f.e.).